src/core/types/elasticsearch/search.ts

```diff
@@ -9,6 +9,11 @@
 import { ValuesType, UnionToIntersection } from 'utility-types';
 import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 
+interface AggregationsAggregationContainer extends Record<string, any> {
```

Comment:
is it done to get rid of `aggregations: unknown` ?

Reply:
yes